examples/porting/lv_port_lcd_stm32_template.c

```diff
@@ -0,0 +1,183 @@
+/**
+ * @file lv_port_lcd_stm32_template.c
+ *
+ * Example implementation of the LVGL LCD display drivers on the STM32 platform
+ */
+
+/*Copy this file as "lv_port_disp.c" and set this value to "1" to enable content*/
+#if 0
+
+/*********************
+ *      INCLUDES
+ *********************/
+/* Include STM32Cube files here, e.g.:
+#include "stm32f7xx_hal.h"
+*/
```

Comment:
```suggestion
#if defined STM32F3
#include "stm32f3xx_hal.h"
#elif defined STM32F4
#include "stm32f4xx_hal.h"
#elif defined STM32F7
#include "stm32f7xx_hal.h"
#elif defined STM32H7
#include "stm32xxx_hal.h" /*Include your hal header */
#endif
```

Reply:
Please note that this code has not been tested on other platforms than the STM32F746. We can add these headers, but what if there are subtle differences between those? Since this is an example, I think it is reasonable to expect that the users will adapt it to their platform.

Reply:
In this case can we add it as an example to one of our lv_port_stm32* repos and link it in the docs?